x-pack/legacy/plugins/code/server/init_es.ts

```diff
@@ -4,17 +4,15 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { Server } from 'hapi';
+import { IClusterClient } from 'src/core/server';
 import { RepositoryIndexInitializerFactory } from './indexer';
 import { RepositoryConfigController } from './repository_config_controller';
 import { EsClientWithInternalRequest } from './utils/esclient_with_internal_request';
 import { EsClient } from './lib/esqueue';
 import { Logger } from './log';
 
-export async function initEs(server: Server, log: Logger) {
-  // wait until elasticsearch is ready
-  await server.plugins.elasticsearch.waitUntilReady();
```

Comment:
we don't need this logic in the legacy platform anymore?

Reply:
This will be a question for you. After we use the `context.elasticsearch.xxx`, I think this is not required. Can you confirm?

Reply:
No, NP doesn't support it yet https://github.com/elastic/kibana/issues/43456